docs/src/modules/components/AppFrame.js

```diff
@@ -205,9 +208,13 @@ function AppFrame(props) {
               data-ga-event-action="language"
             >
               <LanguageIcon />
-            </IconButton>
+              <Hidden xsDown implementation="css">
+                {/* eslint-disable-next-line material-ui/no-hardcoded-labels */}
+                <span className={classes.language}>Language</span>
```

Comment:
I think we want to localize this as well, no?

Reply:
I was proposing the inverse

> What do you think of keeping "Language" hard-coded in English? It could make it easier to find its way back to the correct language.

Any preference?

Reply:
Oh sorry didn't see that. I'm not gonna lie I had this situation a couple of times with other UIs. But then I do know english so finding it in the first place is no problem. I think I prefer localized language when we do have the icon for emergency exits but no strong preference on either one.

Reply:
Right, we have the icon. I haven't considered that.

Reply:
No strong preference either.